spec/lib/extensions/brakeman_fingerprint_patch_spec.rb

```diff
@@ -12,7 +12,22 @@ def location = {}
   shared_examples_for "handles brakeman pathing" do
     before do
       allow(Rails).to receive(:root).and_return(Pathname.new(core_path))
-      allow(described_class).to receive(:rails_engine_paths).and_return([engine_path])
+      allow(Rails::Engine).to receive(:subclasses).and_return([
+        instance_double("Rails::Engine", :root => Pathname.new(engine_path)),
+        instance_double("Rails::Engine", :root => Pathname.new(other_engine_path))
+      ])
+    end
+
+    after do
+      # Clear the rails_engine_paths cache
+      described_class.remove_instance_variable(:@rails_engine_paths)
```

Comment:
Thinking out loud... I believe this is a pretty stable ivar name.  I wish there was an easier way to clear it for testing.

Reply:
I could add it to the EvmSpecHelper.clear_caches thing if you think that's better? https://github.com/ManageIQ/manageiq/blob/master/spec/support/evm_spec_helper.rb#L43

Reply:
I guess we do that generally for things we use elsewhere.  Not sure if it's worth moving it there if this is the only place we use it.  I was more concerned that this is unstable.  This will raise `NameError` if it doesn't exist so I'm fine with it.